osu.Server.Spectator/Hubs/Multiplayer/Matchmaking/Queue/MatchmakingQueueBackgroundService.cs

```diff
@@ -231,6 +235,17 @@ private async Task refreshQueues()
             lastQueueRefreshTime = DateTimeOffset.Now;
         }
 
+        private Task refreshPools()
+        {
+            if (DateTimeOffset.Now - lastPoolRefreshTime < TimeSpan.FromMinutes(5))
+                return Task.CompletedTask;
```

Comment:
Not sure how to feel about this refresh period. I'd increase it significantly, go to like every-hour range or so, probably. Main concern would be the amount of db scraping this is going to do by retrieving all beatmaps.

No idea how the query itself performs in a production setting. At least the select is column-limited to something like (back-of-napkin) 23 bytes per beatmap so maybe it won't return *egregious* amounts of data (but may well be in megabytes range still).